cdc/sink/flowcontrol/flow_control_test.go

```diff
@@ -21,11 +21,16 @@ import (
 	"testing"
 	"time"
 
+	"github.com/stretchr/testify/require"
+
 	"github.com/pingcap/check"
-	"github.com/pingcap/tiflow/pkg/util/testleak"
 	"golang.org/x/sync/errgroup"
 )
 
+func Test(t *testing.T) {
```

Comment:
Should this statement be removed when using the testify framework?

Reply:
Oops! Removed.